### PR TITLE
Docs: Update azuread docs to mention about env variables

### PR DESCRIPTION
**Pull request by ghost**
**What this PR does / why we need it**:
Add a note about the name of the environment variables to be used for configuring client_id and client_secret
It's not clear that the name of the variables should be AZUREAD

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/auth/azuread.md [hunk 1]
```diff
@@ -108,6 +108,12 @@ allowed_domains =
 allowed_groups =
 ```
 
+You can also use the following enviroment variables to configure **client_id** and **client_secret**:
```
Comment (achatterjee-grafana) [inline docs/sources/auth/azuread.md:111]:
```suggestion
You can also use these enviroment variables to configure **client_id** and **client_secret**:
```

File: docs/sources/auth/azuread.md [hunk 2]
```diff
@@ -108,6 +108,12 @@ allowed_domains =
...
+```
+GF_AUTH_AZUREAD_CLIENT_ID
+GF_AUTH_AZUREAD_CLIENT_SECRET
+```
+
```
Comment (achatterjee-grafana) [inline docs/sources/auth/azuread.md:116]:
```suggestion
```

**Review by achatterjee-grafana — COMMENTED** [review]
Minor copy edit suggestions. Also needs a dev review, tagging @joshhunt

**Review by achatterjee-grafana — APPROVED** [review]

**Comment by achatterjee-grafana** [comment]
@sdboyer and @malcolmholmes can one of you review this PR so that we can get this in for 8.1.4. Thank you.

**Review by sdboyer — APPROVED** [review]

**Pull request merged**